docs/install.rst

```diff
@@ -20,49 +20,55 @@ Taichi can be easily installed via ``pip``:
 
+  * **Possible solution**: add ``export TI_USE_UNIFIED_MEMORY=0`` to your ``~/.bashrc``. This disables unified memory usage in CUDA backend.
 
-If you find other CUDA problems:
 
-* Try adding ``export TI_ENABLE_CUDA=0`` to your  ``~/.bashrc``. This disables the CUDA backend completely and Taichi will fall back on other GPU backends such as OpenGL.
+- If you find other CUDA problems:
 
+  * **Possible solution**: add ``export TI_ENABLE_CUDA=0`` to your  ``~/.bashrc``. This disables the CUDA backend completely and Taichi will fall back on other GPU backends such as OpenGL.
 
-If Taichi crashes with a stack backtrace containing a line of ``glfwCreateWindow`` (see `#958 <https://github.com/taichi-dev/taichi/issues/958>`_):
+OpenGL issues
+*************
 
-.. code-block::
+- If Taichi crashes with a stack backtrace containing a line of ``glfwCreateWindow`` (see `#958 <https://github.com/taichi-dev/taichi/issues/958>`_):
 
-    [Taichi] mode=release
-    [E 05/12/20 18.25:00.129] Received signal 11 (Segmentation Fault)
-    ***********************************
-    * Taichi Compiler Stack Traceback *
-    ***********************************
+  .. code-block::
 
-    ... (many lines, omitted)
+        [Taichi] mode=release
+        [E 05/12/20 18.25:00.129] Received signal 11 (Segmentation Fault)
+        ***********************************
+        * Taichi Compiler Stack Traceback *
+        ***********************************
 
-    /lib/python3.8/site-packages/taichi/core/../lib/taichi_core.so: _glfwPlatformCreateWindow
-    /lib/python3.8/site-packages/taichi/core/../lib/taichi_core.so: glfwCreateWindow
-    /lib/python3.8/site-packages/taichi/core/../lib/taichi_core.so: taichi::lang::opengl::initialize_opengl(bool)
+        ... (many lines, omitted)
 
-    ... (many lines, omitted)
+        /lib/python3.8/site-packages/taichi/core/../lib/taichi_core.so: _glfwPlatformCreateWindow
+        /lib/python3.8/site-packages/taichi/core/../lib/taichi_core.so: glfwCreateWindow
+        /lib/python3.8/site-packages/taichi/core/../lib/taichi_core.so: taichi::lang::opengl::initialize_opengl(bool)
 
-This is likely because you are running Taichi on a virtual machine with an old OpenGL. Taichi requires OpenGL 4.3+ to work).
+        ... (many lines, omitted)
 
-* Try adding ``export TI_ENABLE_OPENGL=0`` to your  ``~/.bashrc``, even if you don't initialize Taichi with OpenGL (``ti.init(arch=ti.opengl)``). This disables the OpenGL backend detection to avoid incompatibilities.
+  This is likely because you are running Taichi on a (virtual) machine with an old OpenGL API. Taichi requires OpenGL 4.3+ to work.
 
+  * **Possible solution**: add ``export TI_ENABLE_OPENGL=0`` to your  ``~/.bashrc``, even if you don't initialize Taichi with OpenGL (``ti.init(arch=ti.opengl)``). This disables the OpenGL backend detection to avoid incompatibilities.
```

Comment:
I'd say ''even when you initialize Taichi with other backends than OpenGL'', also I wouldn't use the comma before even.